addons/mail/static/src/xml/systray.xml

```diff
@@ -3,60 +3,140 @@
+            </div>
+        </t>
+        <div t-if="!widget.isMobile()" class="o_mail_systray_dropdown_top">
+            <div>
+                <button type="button" t-attf-class="btn btn-link o_filter_button #{!widget.filter ? 'active' : ''}" role="tab"> All </button>
+                <button type="button" t-attf-class="btn btn-link o_filter_button #{widget.filter === 'chat' ? 'active' : ''}" data-filter='chat' role="tab"> Chat </button>
+                <button type="button" t-attf-class="btn btn-link o_filter_button #{widget.filter === 'channels' ? 'active' : ''}" data-filter='channels' role="tab"> Channels </button>
+            </div>
+            <button type="button" class="btn btn-link o_new_message"> New message </button>
+        </div>
+
+        <t t-set="previews" t-value="widget.previews"/>
+        <t t-call="mail.systray.MessagingMenu.Previews"/>
+
+        <t t-if="widget.isMobile()">
+            <div class="o_mail_mobile_tabs">
+                <div t-attf-class="o_mail_mobile_tab o_filter_button  #{!widget.filter ? 'active' : ''}" role="tab">
+                    <span class="fa fa-envelope"/>
+                    <span class="o_tab_title">All</span>
+                </div>
+                <div t-attf-class="o_mail_mobile_tab o_filter_button #{widget.filter === 'chat' ? 'active' : ''}" data-filter='chat' role="tab">
+                    <span class="fa fa-user"/>
+                    <span class="o_tab_title">Chat</span>
+                </div>
+                <div t-attf-class="o_mail_mobile_tab o_filter_button #{widget.filter === 'channels' ? 'active' : ''}" data-filter='channels' role="tab">
+                    <span class="fa fa-users"/>
+                    <span class="o_tab_title">Channels</span>
+                </div>
+            </div>
+        </t>
+    </t>
+
+    <!--
+        @param {mail.systray.MessagingMenu} widget
+        @param {integer} [widget.counter=0]
+     -->
+    <t t-name="mail.systray.Counter">
+        <t t-if="widget.counter > 0" t-esc="widget.counter"/>
+    </t>
+
     <!--
         @param {Object} previews valid list of objects for preview rendering
           (see mail.Preview template)
     -->
     <t t-name="mail.systray.MessagingMenu.Previews">
-         <t t-if="_.isEmpty(previews)">
-            <li class="text-center o_no_activity mt16">
-                <span>No conversation yet...</span>
-            </li>
-        </t>
-        <t t-foreach="previews" t-as="preview">
-            <t t-call="mail.Preview"/>
-        </t>
+        <div class="o_mail_systray_dropdown_items">
+            <t t-if="_.isEmpty(previews)">
+                <t t-call="mail.systray.MessagingMenu.EmptyPreviews"/>
+            </t>
+            <t t-foreach="previews" t-as="preview">
+                <t t-call="mail.Preview"/>
+            </t>
+        </div>
+    </t>
+
+    <t t-name="mail.systray.MessagingMenu.EmptyPreviews">
```

Comment:
Dedicated template, because module `mail_bot` will replace, so that static previews do not display this.

Without replacement of this template by `mail_bot` module:
```
+------------------------+
| No conversation yet... |
+------------------------+
|     OdooBot notif      |
+------------------------+
```

With replacement of this template by `mail_bot` module:
```
+------------------------+
|     OdooBot notif      |
+------------------------+